Doc/library/asyncio-task.rst

```diff
@@ -225,6 +225,18 @@ Running an asyncio Program
     the end.  It should be used as a main entry point for asyncio
     programs, and should ideally only be called once.
 
+    Return a result of *coro* execution, or raise a :exc:`RuntimeError`
+    if ``asyncio.run()`` is called from a running event loop, or a
+    :exc:`ValueError` if *coro* is not a courutine.
```

Comment:
Thanks for the adding the additional information @eamanu, but I noticed a couple of issues with the section
added. 

Noticed a spelling typo here, "courutine" should be "coroutine". Also, it's helpful to link to the glossary using:
```
:term:`coroutine`
```

Also, while this section is helpful (particularly with specifying the `RuntimeError` and `ValueError` scenarios), I think it could be phrased significantly better. Instead, I would suggest the following:

```
Within the new event loop, *coro* is executed, returning the result. If
``asyncio.run()`` is called within a running event loop, a :exc:`RuntimeError`
is raised. If *coro* is not a coroutine, a :exc:`ValueError` is raised.
```

Personally, I think it's especially important to phrase this section of the documentation as well as possible, since it's likely one of the most read sections for the entire asyncio module.

Let me know what you think @asvetlov and @1st1. If you approve of the suggestions, I'll open a new PR that
makes the changes I mentioned above in the docs for `ascynio.run()` and in the docstring as well. 

Grammar tip: When chaining together multiple instances of "or", each of the items should be separated by
a comma, but only the last one is supposed to have the "or". For example: "item1, item2, or item3" instead of
"item1, or item2, or item3". The section added to the documentation in this PR used the latter.

Reply:
I'm not sure I follow "Within the new event loop, *coro* is executed, returning the result." -- I can't parse it.  I'd keep the original one "Return the result of *coro* execution."

IMO it's not necessary to document RuntimeError and ValueError; "This function cannot be called when another asyncio event loop is running in the same thread." already covers that.

Reply:
@1st1 Okay, I'll make the suggested changes in https://github.com/python/cpython/pull/16403.